main.go

```diff
@@ -68,6 +70,15 @@ func main() {
 		log.Fatal(err)
 	}
 
+	// Create a done channel.
+
+	doneCh := make(chan struct{})
+	defer close(doneCh)
+	userList = make(map[string]struct{})
+	for userObj := range mc.ListObjects(userBucket, "", true, doneCh) {
```

Comment:
is the list objects function will keep going to put objects to userObj chan when there has a new item unless the doneCh closed?
please dig into see how it implemented. :bow

Reply:
@tommady 
Thanks for your review.
https://github.com/minio/minio-go/blob/79f99bc54ad036fd2ef9166149e9a8de33920edc/api-list.go#L431
The goroutine will end after the result is received.

Reply:
cool awesome, then we need to find a way to trigger reload user bucket with multiple pods.